### DISCUSSION TITLE
Still show issues after archiving

### DISCUSSION THREAD

**Discussion by retrievaaaa** [community/community#13711] [Projects and Issues]
Hello,
I think still being able to look at issues after a repository has been archived would be very useful. A lot of times, you can find fixes to your problems / the cause to your problem in the issues. They should be locked, but maybe still visible, since a lot of valuable information can be in there.

**Comment by driesvints** [comment]
This is already the case right now? Here's an issue from a public archived repo: https://github.com/laravel/ideas/issues/2667
